check_synology.py

```diff
@@ -68,7 +81,7 @@ def exitCode():
     if critical and critical < int(math.ceil(float(load1))):
         state = 'CRITICAL'
 
-    print state + ' - load average: %s, %s, %s' % (load1, load5, load15), '| load1=%sc' % load1, 'load5=%sc' % load5, 'load15=%sc' % load15
+    print(state + ' - load average: %s, %s, %s' % (load1, load5, load15), '| load1=%sc' % load1, 'load5=%sc' % load5, 'load15=%sc' % load15)
```

Comment:
That change now results in:
```('OK - load average: 0.92, 1.24, 1.44', '| load1=0.92c', 'load5=1.24c', 'load15=1.44c')```
We need to remove the parentheses otherwise icinga will include it into the output